src/components/ui/ListItem.svelte

```diff
@@ -1,10 +1,16 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
+  import type { Action } from 'svelte/action';
 
   export { className as class };
   export { styleName as style };
   export let draggable: boolean = false;
 
+  export let tooltip: Action<HTMLElement, any> | undefined = undefined;
+  export let tooltipContent: any = undefined;
```

Comment:
Try to avoid using `any` when possible. This is more likely to be a `string | undefined`